src/system-plugins/software-update-alert/public/webcomponents/orov-software-update-router.html

```diff
@@ -24,6 +24,8 @@
               status.progress;
 
               switch(status.progress){
+                  case "Examining":
+                  break;
```

Comment:
Will JS be sad if this is not tabbed underneath the previous case? Python would be! :smile: